src/Appwrite/CLI/TasksService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Appwrite\CLI;
+
+use Utopia\Platform\Service;
+use Appwrite\CLI\Tasks\Doctor;
+use Appwrite\CLI\Tasks\Install;
+use Appwrite\CLI\Tasks\Maintenance;
+use Appwrite\CLI\Tasks\Migrate;
+use Appwrite\CLI\Tasks\SDKs;
+use Appwrite\CLI\Tasks\Specs;
+use Appwrite\CLI\Tasks\SSL;
+use Appwrite\CLI\Tasks\Usage;
+use Appwrite\CLI\Tasks\Vars;
+use Appwrite\CLI\Tasks\Version;
+use VolumeSync;
+
+class TasksService extends Service
+{
+    public function __construct()
+    {
+        $this->type = self::TYPE_CLI;
+        $this
+            ->addAction(Version::getName(), new Version())
```

Comment:
Rethinking about this. Do we still need the name if it's part of the interface? Can't we just take it inside the constructor?

Reply:
If we do that. I think we can change getName to not be a static class.

Reply:
what do we do? move getName as instance function? that will have to be done in the `Action` and we need the name in order to reference the action, later to remove action?
Not sure I fully understand this